docs/reference/config.md

```diff
@@ -78,6 +79,8 @@ Each package document has the following keys:
   - Customize the name of the querier file. Defaults to `querier.go`.
 - `output_files_suffix`:
   - If specified the suffix will be added to the name of the generated files.
+- `query_parameter_limit`:
+  - Positional arguments that will be generated in go functions (>= `1` or `-1`). To always emit a parameter struct, you would need to set it to `-1`. `0` is invalid. Defaults to `1`.
```

Comment:
I changed my mind here. We should allow for zero to be a valid value. Anything less than zero should be an error.